packages/zoltan2/src/input/Zoltan2_PamgenMeshAdapter.hpp

```diff
@@ -95,6 +95,7 @@ template <typename User>
   typedef typename InputTraits<User>::scalar_t scalar_t;
   typedef typename InputTraits<User>::lno_t    lno_t;
   typedef typename InputTraits<User>::gno_t    gno_t;
+  typedef typename InputTraits<User>::lno_t offset_t; // use lno_t
```

Comment:
I know I said to make this default assignment to offset_t, but I wonder now whether this decision will come back to burn us later (e.g., a user defines offset_t != lno_t in InputTraits and then wonders why his offset_t isn't being used).  
Would there be any harm in using InputTraits<User>::offset_t for offset_t here?  

Our PamgenMeshAdapter tests (in Zoltan2_Typedefs.hpp, pamgenMeshAdapterTest.cpp, PamgenMeshInput.cpp, GraphModel2ndAdjsFromAdjs.cpp) currently use TpetraMultiVector as their User type.  I've always thought that was an unusual choice but it never impacted anything before.  With offset_t redefined, it probably makes more sense for our PamgenMeshAdapter tests to use BasicUserTypes as its User type.